ledger-tool/src/main.rs

```diff
@@ -1918,6 +1918,7 @@ fn main() {
             .arg(&no_snapshot_arg)
             .arg(&account_paths_arg)
             .arg(&accounts_db_skip_initial_hash_calc_arg)
+            .arg(&accountsdb_skip_shrink)
```

Comment:
it looks like this speeds up a bit? Feature parity with other `solana-ledger-tool` subcommands.

Reply:
Yeah probably a good option to allow - nice.